### ISSUE TITLE
Investigate Database-Server (Architecture) Documentation

### ISSUE THREAD

**Issue by holtzmak** [holtzmak/Humane-Transport#40]
This issue is split from #27 .

Investigate ways we could document our expected Database-Server architecture/design:
1. We could start drawing Database-Server Architecture Entity Diagrams to make our table definition.
1. What about other/more ways to document database building besides making an entity diagram for our problem?

Create a Wiki page with your findings (and maybe start the UML Diagrams depending on our progress with the animal transport form breakdowns)

**Comment by holtzmak** [comment]
Should also look into how images, etc. could be stored/represented in the database. Do images need special treatment in our documentation?

**Comment by mansiPaTeLL** [comment]
I've investigated the ways to create data model for the database. Based on investigation, I found that we should use relational database for our application. It will help us to organize data much better as we don't large amount of users. I have found ways to create data model to implement database. Refer the WikiPage -  [Data Modelling Design Process](https://github.com/holtzmak/Humane-Transport/wiki/Data-Modelling-Design-Process)

In this process, We need to create the "Entity/Attribute List" (First Step) - I've created the first draft of "Entity/Attribute List" - See [Entity/Attribute List](https://docs.google.com/document/d/1HA8zxGMal3F3KFdn4tdBiznNW4NaCpU7zrRbOdseNj0/edit?usp=sharing)

We can use the 3-tier Database Architecture for presenting, managing and storing/retrieving data. The 3-trie architecture includes: Presentation tier, Application Tier, Data tier. This architecture works well with Flutter. 

Now, my next task is to test the different database servers using flutter aa well as how to store/retrieve can be possible using Flutter with the selected database.

**Comment by holtzmak** [comment]
Your first draft at the entity/attribute is a really good start! Same with your wiki page, both are really well thought out.

> Now, my next task is to test the different database servers using flutter aa well as how to store/retrieve can be possible using Flutter with the selected database.

A new task would be good for this, and would be blocked by #56 - once we have the framework in you can test to your heart's content! I'd like to mention that SQLite is a good place to start investigating how an implementation works, because we've seen that it's the favorite with mobile applications... In fact, a good google search should probably answer most questions!

**Comment by mansiPaTeLL** [comment]
@holtzmak Yes, I thought the same. The SQLite can be good database for our application. Thank you to review this.
